### ISSUE TITLE
Are there any detailed API documentations of Lottie?

### ISSUE THREAD

**Issue by wangjiangchuan** [airbnb/lottie-android#403]
Before opening a new issue:

* Look through open and existing issues for duplicates.
* Apply an appropriate label.

If this is a bug:
* Attach your After Effects file. If you do not feel comfortable posting it on github, email 
it to lottie@airbnb.com. There are 500 ways to do everything in After Effects that achieves the 
same visual effect so it is nearly impossible to reproduce your issue without the original After 
Effects file. Github doesn't allow you to attach a json or aep file directly so compress it and attach it as a zip file.

If this is a feature request:
* Explain what feature you would like and why you need it. There are usually After Effects 
workarounds that we can provide in the meantime that achieve the same effect.
* What version of Lottie did you test this on? Check the readme for the latest version and makes sure it hasn't been resolved.

**Comment by gpeal** [comment]
@wangjiangchuan Keep an eye on [http://airbnb.io/lottie/](http://airbnb.io/lottie/) over the coming weeks :)

**Comment by wangjiangchuan** [comment]
Thanks, I really get some help from these sites. However, I am a android engineer,  so what I want is some API documents, because I had used Lottie 1.1 in my projects and now Lottie is 2.1. Some some code in my project can not play the animation created from AE and even throw exception. But when I debug the problem, I founds there are some public interfaces provides by Lottie without any documentation. So I can't figure out the best way to use the API. Never mind, we used some special way to reflect to the protected API to avoid lost of changes in code. Never mind, I still appreciate a lot.
